### PR TITLE
K8SPG-233 remove unneeded options from 'operator.yaml'

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/init-deploy.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/scheduled-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/upgrade.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/version-service.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/demand-backup.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/smart-update.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/210f060e/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/297/commits/210f060e649779bd5dc5863fdc36c1fea461220a
image: `perconalab/percona-postgresql-operator:PR-297-210f060e-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-297-210f060e-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-297-210f060e-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-297-210f060e-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-297-210f060e-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-297-210f060e-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/init-deploy.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/upgrade.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/version-service.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/scheduled-backup.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/demand-backup.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/scaling.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/smart-update.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/297/commits/1ff4cc0f0ceceb784c91880a7123401b4720588e
image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/init-deploy.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/upgrade.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/scaling.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/smart-update.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/operator-self-healing.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/scheduled-backup.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/version-service.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/ns-mode.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/1ff4cc0f/demand-backup.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/297/commits/1ff4cc0f0ceceb784c91880a7123401b4720588e
image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-297-1ff4cc0f-pgo-deployer`

**Review by egegunes — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/init-deploy.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/demand-backup.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/version-service.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/upgrade.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/scheduled-backup.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/smart-update.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/9acb6df2/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/297/commits/9acb6df2cfd342c0faebab61f5450d28f500b275
image: `perconalab/percona-postgresql-operator:PR-297-9acb6df2-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-297-9acb6df2-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-297-9acb6df2-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-297-9acb6df2-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-297-9acb6df2-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-297-9acb6df2-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| version-service | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/version-service.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/demand-backup.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/scheduled-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/upgrade.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/init-deploy.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/smart-update.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/22ca2a06/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/297/commits/22ca2a060682750c7beb89f13947e4282b8b4dec
image: `perconalab/percona-postgresql-operator:PR-297-22ca2a06-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-297-22ca2a06-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-297-22ca2a06-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-297-22ca2a06-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-297-22ca2a06-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-297-22ca2a06-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/scheduled-backup.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/version-service.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/upgrade.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/init-deploy.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/smart-update.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/a2f2419a/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/297/commits/a2f2419a3948dca79c8c853fa343b99cc249d7e1
image: `perconalab/percona-postgresql-operator:PR-297-a2f2419a-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-297-a2f2419a-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-297-a2f2419a-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-297-a2f2419a-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-297-a2f2419a-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-297-a2f2419a-pgo-deployer`

File: deploy/operator.yaml [hunk 1]
```diff
@@ -276,7 +167,7 @@ spec:
       restartPolicy: Never
       containers:
         - name: pgo-deploy
-          image: perconalab/percona-postgresql-operator:main-pgo-deployer
+          image: perconalab/percona-postgresql-operator:update-options-pgo-deployer
```
Comment (hors) [inline deploy/operator.yaml:170]:
```suggestion
          image: perconalab/percona-postgresql-operator:main-pgo-deployer
```

**Review by cap1984 — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/init-deploy.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/version-service.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/scheduled-backup.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/upgrade.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/smart-update.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/recreate.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/affinity.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/monitoring.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/self-healing.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/operator-self-healing.log) |
| clone-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/clone-cluster.log) |
| tls-check | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/tls-check.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/users.log) |
| ns-mode | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-297/8d351957/ns-mode.log) |
| We run 16 out of 16|

commit: https://github.com/percona/percona-postgresql-operator/pull/297/commits/8d351957bb76f72350998cc535a396fc2e3fd10f
image: `perconalab/percona-postgresql-operator:PR-297-8d351957-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-297-8d351957-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-297-8d351957-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-297-8d351957-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-297-8d351957-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-297-8d351957-pgo-deployer`

**Pull request merged**